consensus/common_test.go

```diff
@@ -37,6 +37,23 @@ func ensureDir(dir string, mode os.FileMode) {
 	}
 }
 
+type NodeConfig struct {
+	cfg.Config `mapstructure:",squash"`
+	P2P        *p2p.Config   `mapstructure:"p2p"`
+	Mempool    *mempl.Config `mapstructure:"mempool"`
+	Consensus  *Config       `mapstructure:"consensus"`
+}
+
+// TODO: This is the same as NodeConfig. Should we move the configs to types (?)
```

Comment:
Okay, maybe everyone will hate me now, but... (with a very brief review of the code)...

What about putting all the config files together in `types/config.go` and just import that.  They seem to be needed in multiple packages, and data that is passed from package to package all seems to go in `types`.

If nothing else, they are needed by command package to parse into, and the other packages to configure themselves.